### PR TITLE
Add e2e tests for cert_manager integration

### PR DESCRIPTION
**Pull request by arapulido**
### What does this PR do?

Adds e2e using kind for cert_manager integration

### Motivation

cert_manager didn't have e2e tests, but the addition of utilities to create test environments for Kubernetes using kind, now makes this a lot easier.

This PR depends on this fix to be released: https://github.com/DataDog/integrations-core/pull/8783

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

**Comment by arapulido** [comment]
E2E tests failing is expected, as we need to wait for this https://github.com/DataDog/integrations-core/pull/8783 to be released

**Comment by arapulido** [comment]
Moving this PR to draft, until we have a new release of `datadog_checks/dev`

**Review by hithwen — APPROVED** [review]

File: cert_manager/datadog_checks/cert_manager/cert_manager.py [hunk 1]
```diff
@@ -1,16 +1,20 @@
-# (C) Datadog, Inc. 2019
+# (C) Datadog, Inc. 2019-2021
```
Comment (hithwen) [inline cert_manager/datadog_checks/cert_manager/cert_manager.py:1]:
```suggestion
# (C) Datadog, Inc. 2019-present
```

Comment (arapulido) [reply cert_manager/datadog_checks/cert_manager/cert_manager.py]: Added alongside another one

**Pull request merged**